src/main/kotlin/com/sourcegraph/cody/config/CodyAccountDetailsProvider.kt

```diff
@@ -23,15 +23,15 @@ class CodyAccountDetailsProvider(
   ): CompletableFuture<DetailsLoadingResult<CodyAccountDetails>> {
 
     return ProgressManager.getInstance()
-        .submitIOTask(indicator) { indicator ->
+        .submitIOTask(indicator) {
```

Comment:
Was than any error?
Previous version was more readable.

Reply:
The compiler was complaining because the argument of the lambda expression shadows the method argument.